packages/back-end/src/scim/scim.router.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from "express";
+import bodyParser from "body-parser";
+import authenticateApiRequestMiddleware from "../middleware/authenticateApiRequestMiddleware";
+import usersRouter from "./users/users.router";
+import scimMiddleware from "./middleware/scimMiddleware";
+import groupsRouter from "./groups/groups.router";
+
+const router = Router();
+
+router.use(bodyParser.json({ limit: "1mb" }));
+router.use(bodyParser.urlencoded({ limit: "1mb", extended: true }));
```

Comment:
Do we need these? We already set the bodyParser one layer above in the `app.ts` file